style.css

```diff
@@ -23,28 +23,29 @@
   background-color: #616161;
 }
 
-.burger:before {
+.burger::before {
   top: -8px;
 }
 
-.burger:after {
+.burger::after {
   bottom: -8px;
 }
 
-@media (max-width:400px) {
-.content__description { font-size: 3px; } 
-   }
-    @media (min-width:401px) and (max-width:599px) { .content__description { font-size: 6px; }
-}
-@media (min-width:600px) and (max-width:767px) { 
-.content__description { font-size: 9px; } 
-}
-@media (min-width:768px) and (max-width:950px) { 
-.content__description { font-size: 12px; }
-}
-@media (min-width:951px) and (max-width:1050px) {
-.content__description { font-size: 15px; } 
+@media (max-width: 700px) {
+  .content__description { font-size: 13px; }
+
+  .menu__link {
+    display: none;
+  }
+
+  .burger {
+    display: block;
+    top: 10px;
+    left: 10px;
+    cursor: pointer;
+  }
 }
-@media (min-width:1051px) { 
-  .content__description { font-size: 18px; } 
-}    
\ No newline at end of file
+
+.content {
+  padding-top: 10px;
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/HelenRom3/pullRequest?prid=6002362)